index.js

```diff
@@ -43,6 +43,7 @@ module.exports = {
                 "qunit/no-global-expect": "error",
                 "qunit/no-global-module-test": "error",
                 "qunit/no-global-stop-start": "error",
+                "qunit/no-hooks-from-ancestor-modules": "error",
```

Comment:
As noted in another comment, we can't make changes to the exported configs until a major release. Also, I think this makes sense to include in both "recommended" and "two" (in the next major release). But for this PR, please remove this line (unless you are okay with this going in the next major release).

Reply:
updated!